packages/sit-onyx/src/styles/grid.scss

```diff
@@ -148,6 +175,10 @@ $sidebar-breakpoints: (
   * SIDEBAR GRID
   */
   .onyx-sidebar {
+    .onyx-grid-layout {
+      padding: var(--onyx-density-md) var(--onyx-grid-margin) var(--onyx-density-md)
+        var(--onyx-grid-margin);
```

Comment:
```suggestion
      padding: var(--onyx-density-md) var(--onyx-grid-margin);
```

Reply:
Open question: Should we add a new CSS variable for the top/bottom padding so its easier to change it if needed?
Maybe something like `--onyx-grid-margin-vertical`?